plugins/kotlin-as-java/src/main/kotlin/jvmOverloads.kt

```diff
@@ -0,0 +1,12 @@
+package org.jetbrains.dokka.kotlinAsJava
+
+import org.jetbrains.dokka.model.Annotations
+import org.jetbrains.dokka.model.Documentable
+import org.jetbrains.dokka.model.properties.WithExtraProperties
+import org.jetbrains.kotlin.utils.addToStdlib.firstNotNullResult
+
+internal fun WithExtraProperties<out Documentable>.jvmOverloads(): Annotations.Annotation? =
+    extra[Annotations]?.directAnnotations?.entries?.firstNotNullResult { (_, annotations) -> annotations.jvmOverloadsAnnotation() }
+
+internal fun List<Annotations.Annotation>.jvmOverloadsAnnotation(): Annotations.Annotation? =
```

Comment:
Since this is not used in any other place than the line before, I'd make it private, make it return `Boolean` and call it `hasJvmOverloads`; or alternatively just create a method `isJvmOverloads` that would check if an annotation is a `@JvmOverloads` one and use `annotations.any { it.isJvmOverloads() }` in line 9. 

It is functionally equivalent but just looks better IMHO